cmd/talosctl/acompat/acompat.go

```diff
@@ -0,0 +1,14 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package acompat provides compatibility with gRPC 1.67.0 and later.
+package acompat
+
+import "os"
+
+func init() {
+	if err := os.Setenv("GRPC_ENFORCE_ALPN_ENABLED", "false"); err != nil {
```

Comment:
do we need that given that we don't bump gRPC?

Reply:
Yeah, the env variable is already there, it's just not enabled by [default on 1.66.3](https://github.com/grpc/grpc-go/blob/v1.66.3/internal/envconfig/envconfig.go#L48). So I think it's good idea to have this variable.